### DISCUSSION TITLE
[2021.08.26] 듣도 보도 못한 사람

### DISCUSSION THREAD

**Discussion by chaeyeon4u** [pyunji/Hyundai-ITnE-Team_QuadCore#58] [Algorithm]
[듣도 보도 못한 사람](https://www.acmicpc.net/problem/1764)

**Comment by hyk339** [comment]
[듣도 보도 못한 사람 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/baekjoon/BOJ_1764.java)

**Comment by pyunji** [comment]
[듣보잡 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/BOJ/BOJ_1764.java)

**Comment by leechang4** [comment]
[듣도 보도 못한 사람 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/lch/BOJ/BOJ%201764.java)

**Comment by chaeyeon4u** [comment]
[듣도 보도 못한 사람 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/BOJ_1764.java)
